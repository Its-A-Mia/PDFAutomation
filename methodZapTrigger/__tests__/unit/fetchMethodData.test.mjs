import { fetchMethodData } from '../../index.mjs';

const snapshot = {
  count: expect.any(Number),
  value: expect.any(Array),
};

describe('fetchMethodData function', () => {
  it('grabs invoice entries with a balance due', async () => {
    const data = await fetchMethodData();
    expect(data).toMatchSnapshot(snapshot);
  });
});
