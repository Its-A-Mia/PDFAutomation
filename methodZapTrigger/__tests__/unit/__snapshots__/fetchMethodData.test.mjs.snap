// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fetchMethodData function grabs invoice entries with a balance due 1`] = `
{
  "count": Any<Number>,
  "value": Any<Array>,
}
`;
